src/components/RewardCard.jsx

```diff
@@ -0,0 +1,163 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import ReactCardFlip from 'react-card-flip'
+
+const Container = styled.div`
+  background-image: ${p => p.theme.colors.backgroundTertiary};
+  border-radius: 8px;
+  padding: 20px;
+  width: 280px;
+  height: 160px;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+`
+
+const Grid = styled.div`
+  display: grid;
+  grid-template-columns: 1fr 2fr;
+`
+
+const Title = styled.h1`
+  font-size: 24px;
+  font-weight: 400;
+  color: ${p => p.theme.colors.lines};
+`
+
+const Text = styled.h3`
+  font-size: 18px;
+  font-weight: 300;
+  margin-top: -12px;
+  color: ${p => p.theme.colors.lines};
+`
+
+const Description = styled.h3`
+  font-size: 18px;
+  font-weight: 300;
+  text-align: center;
+  color: ${p => p.theme.colors.lines};
+`
+
+const SubTitle = styled.p`
+  font-size: 14px;
+  font-weight: 700;
+  color: ${p => p.theme.colors.bar};
+  text-align: right;
+  margin-top: 8px;
+`
+
+const Icon = styled.img`
+  width: 70px;
+  height: 100%;
+  object-fit: contain;
+`
+
+const InfoIcon = styled.div`
+  position: absolute;
+  top: 16px;
+  right: 16px;
+  width: 24px;
+  height: 24px;
+  border: 2px solid ${p => p.theme.colors.lines};
+  border-radius: 50%;
+  color: ${p => p.theme.colors.lines};
```

Comment:
this one too!

Reply:
oh it looks like theme.colors.lines and theme.colors.text both have the same hexcode of #45171A, should I just change it to text so it's more consistent or is it fine to leave it?

Reply:
yeah i see why that was confusing LOL 

it would be amazing if you could change it to text so it matches up with the variable being used in figma! in case those two variables are different for future reskins

Reply:
i'll write the docs for this sometime it's my bad for not specifying it sorry

Reply:
ok sounds good! if i push another change do you know if i have to open a new PR because this one was merged?

Reply:
yepp i can help merge it in